src/main/java/com/prgrms/needit/domain/user/member/entity/Member.java

```diff
@@ -109,8 +109,10 @@ public void changeMemberInfo(
 		this.introduction = introduction;
 	}
 
-	public void addFavCenter(Center center) {
-		this.favoriteCenters.add(buildFavCenter(center));
+	public FavoriteCenter addFavCenter(Center center) {
+		FavoriteCenter favCenter = buildFavCenter(center);
+		this.favoriteCenters.add(favCenter);
```

Comment:
```suggestion
		this.favoriteCenters.add(buildFavCenter(center));
```
이렇게 한줄로 작성할 줄 있을 것 같네요!

Reply:
중복해서 등록되는 오류를 수정하면서 생성된 favCenter를 return 했어야 해서 따로 variable로 뒀습니다! 
안 그래도 addFavCenter > buildFavCenter > createFavCenter > favCenter 생성자가 있어서 생성된 favCenter를 반환해서 사용해야할 때 어떡해야할지 고민이었는데 이 부분도 리팩토링하면서 생각해봐야할 것 같아요!